### PR TITLE
Update documentation examples to use es6 syntax

### PR DESCRIPTION
**Pull request by Aapzu**
More street cred for using es6😎

### CODE REVIEW

**Comment by hpihkala** [comment]
Hmm - why give an example that has different compatibility level as the distributed code?

I mean, we transpile our es6 code to es5 so that it would work on more browsers, but then we give a usage example in es6? Isn't that a contradiction?

**Comment by Aapzu** [comment]
@hpihkala in a way that's true, of course. But `let`, `const` and arrow functions are such basic features (work in all browsers except IE) that I'd see that not using them is much worse (gives the impression that we are fossil ourselves) than using them even though the code wouldn't work directly in IE anymore

File: README.md [hunk 1]
```diff
@@ -163,14 +163,14 @@ The client and the subscriptions can fire events as detailed below. You can bind
...
 	    console.log('Yeah, we are connected now!')
 	})
 
     // So does the Subscription object
-	var sub = client.subscribe(...)
-	sub.on('subscribed', function() {
-		console.log('Subscribed to '+sub.streamId)
+	const sub = client.subscribe(...)
+	sub.on('subscribed', () => {
+	    console.log('Subscribed to '+sub.streamId)
```
Comment (hpihkala) [inline README.md]: Template literal

**Comment by hpihkala** [comment]
I'm ok with updating examples to something that fail on IE but work in modern browsers. This seems to be the case for const/let, arrow functions and template literals.

File: README.md (hunk 1 above)
Comment (Aapzu) [reply README.md]: Oops, didn't notice that one

**Comment by Aapzu** [comment]
@hpihkala updated the forgotten template literal

**Pull request merged**